tasks/jasmine-chromeapp.js

```diff
@@ -57,7 +57,7 @@ module.exports = function (grunt) {
 
     // Copy user files.
     if (!ctx.paths) {
-      ctx.paths = grunt.file.expand(ctx.files);
+      ctx.paths = grunt.file.expand(ctx.files[0].src);
```

Comment:
confused by this. Wouldn't this cause only "freedom-for-chrome.js" to be whitelisted in this example: https://github.com/freedomjs/freedom-for-chrome/blob/master/Gruntfile.js#L107

Maybe we need to concatenate the expansion of all the sources in `ctx.files`.